docs/intro/contributing.txt

```diff
@@ -199,8 +199,9 @@ work and haven't broken other parts of Django. If you've never run Django's test
 suite before, it's a good idea to run it once beforehand to get familiar with
 its output.
 
-Before running the test suite, install its dependencies by ``cd``-ing into the
-Django ``tests/`` directory and then running:
+Before running the test suit, enter django “tests/” directory by running the command 
+“cd tests” and then install its dependencies by running:
```

Comment:
We should use double backticks, `Django` should be capitalized, also `the` is missing:
```suggestion
Before running the test suit, enter the Django ``tests/`` directory using the
``cd tests`` command, and install test dependencies by running:
```

Reply:
ok, I will take care of these. Thank you to guide me